python_modules/dagster/dagster/_core/execution/plan/plan.py

```diff
@@ -979,7 +979,7 @@ def build_memoized_plan(
                     resources=resources,
                     version=step_output_versions[step_output_handle],
                 )
-                if not io_manager.has_output(context):
+                if not io_manager.has_output(context):  # TODO - this is the problem.
```

Comment:
Need to update has_output to check for the tag so that assets are supported for memoization